apps/vmq_server/src/vmq_reg_redis_trie.erl

```diff
@@ -136,6 +136,18 @@ fold_local_shared_subscriber_info(
         SubscriberId, SubscribersList, FoldFun, FoldFun(SubscriberInfo, SubscriberId, Acc)
     ).
 
+set_complex_trie_version_metrics(File) ->
+    case vmq_util:extract_version(File) of
+        Version when is_list(Version) ->
+            vmq_metrics:update_config_version_metric({complex_trie_version, [{version, Version}]});
```

Comment:
Pass only Version

Reply:
Updated the fn to take Metric name and Version as arguments.